src/Components/Samples/BlazorUnitedApp/Pages/FetchData.razor

```diff
@@ -1,47 +1,9 @@
 ﻿@page "/fetchdata"
-@using BlazorUnitedApp.Data
-@inject WeatherForecastService ForecastService
 
 <PageTitle>Weather forecast</PageTitle>
 
 <h1>Weather forecast</h1>
 
 <p>This component demonstrates fetching data from a service.</p>
 
-@if (forecasts == null)
-{
-    <p><em>Loading...</em></p>
-}
-else
-{
-    <table class="table">
-        <thead>
-            <tr>
-                <th>Date</th>
-                <th>Temp. (C)</th>
-                <th>Temp. (F)</th>
-                <th>Summary</th>
-            </tr>
-        </thead>
-        <tbody>
-            @foreach (var forecast in forecasts)
-            {
-                <tr>
-                    <td>@forecast.Date.ToShortDateString()</td>
-                    <td>@forecast.TemperatureC</td>
-                    <td>@forecast.TemperatureF</td>
-                    <td>@forecast.Summary</td>
-                </tr>
-            }
-        </tbody>
-    </table>
-}
-
-@code {
-    private WeatherForecast[]? forecasts;
-
-    protected override async Task OnInitializedAsync()
-    {
-        forecasts = await ForecastService.GetForecastAsync(DateOnly.FromDateTime(DateTime.Now));
-    }
-}
+<ForecastTable />
```

Comment:
Is there some limitation that makes this necessary? What would happen if the table was inlined into `FetchData.razor`, and that was marked with `@attribute [RenderModeServer]`?

Reply:
Nope, this isn't necessary - that scenario works fine. I was just modifying the sample project to validate various scenarios, and the changes will be reverted before this gets merged.